src/components/DisplayFileCards.tsx

```diff
@@ -63,6 +67,17 @@ export const DisplayFileCards = ({
 				deleteContact={deleteContact}
 			/>
 		);
+	if (index === 3)
+		return (
+			<FileCards
+				files={[]}
+				programs={myPrograms}
```

Comment:
Could be better to really split the two parts with the current FileCard taking only files and a ProgramCard taking programs instead of passing empty arrays imho (it could also allow a slightly different design to better recognize Programs and Files)

Reply:
We thought of making specific cards for the programs. The problem is that apart from the file type nothing changed. As a result, we found it irrelevant to duplicate the code to just change the file type.

Reply:
> We thought of making specific cards for the programs. The problem is that apart from the file type nothing changed. As a result, we found it irrelevant to duplicate the code to just change the file type.

Is it possible to update a program content in Aleph ?

Because our PR #61 adds file change (name and/or content), for name update it shouldn't be a big problem to support programs because we only interact with the post message, but for the content change we are using a Store message so we'll need to use a Program one if needed.

Reply:
I agree with @RezaRahemtola, I'm not ok with the *"it's the same content as the file one"* but that's another question, the real question is if in the future you want to add some feature specific to a program or change the display of theses cards, you will need to create a new card and change the code to the new card and change again the `FileCards` component to his older version. That's just a waste of time.

Another thing that makes me not really a fan about this change, is that a `FileCards` like it's name said, it's for the **files** not the programs.

And I'm really sorry but this is not true:
> The problem is that apart from the file type nothing changed.

Yes for the data it's the same thing, well not really because you have two types one for the files: `IPCFile` and one for the programs `IPCProgram` but ok maybe for the display the data are the same.
There is still a **BIG** problem with that because the `FileCards` component print both the files and the programs AND the program have not the same display than the file one.

So to conclude, it's totatly useless and problematic to merge in the `FileCards` both the files and the programs display. 

Because you use the same component for something that at first glance is the same, but in reality not at all.
They are both different on the data, the display, the place where they are displayed and the way each types works.

:information_source: To know if you can use an already created component, you need to ask you a few questions:
- The first one, is it the same data/ type?
- The second one, need I to change the actual component?
If you answer no to the first one or yes to the second one, unfortunately you can't use this component for your using.

Reply:
> I agree with @RezaRahemtola, I'm not ok with the _"it's the same content as the file one"_ but that's another question, the real question is if in the future you want to add some feature specific to a program or change the display of theses cards, you will need to create a new card and change the code to the new card and change again the `FileCards` component to his older version. That's just a waste of time.
> 
> Another thing that makes me not really a fan about this change, is that a `FileCards` like it's name said, it's for the **files** not the programs.
> 
> And I'm really sorry but this is not true:
> 
> > The problem is that apart from the file type nothing changed.
> 
> Yes for the data it's the same thing, well not really because you have two types one for the files: `IPCFile` and one for the programs `IPCProgram` but ok maybe for the display the data are the same. There is still a **BIG** problem with that because the `FileCards` component print both the files and the programs AND the program have not the same display than the file one.
> 
> So to conclude, it's totatly useless and problematic to merge in the `FileCards` both the files and the programs display.
> 
> Because you use the same component for something that at first glance is the same, but in reality not at all. They are both different on the data, the display, the place where they are displayed and the way each types works.
> 
> information_source To know if you can use an already created component, you need to ask you a few questions:
> 
> * The first one, is it the same data/ type?
> * The second one, need I to change the actual component?
>   If you answer no to the first one or yes to the second one, unfortunately you can't use this component for your using.

At the beginning, I had two additional files : ``ProgramCards.tsx`` and ``ProgramCard.tsx`` for the programs. But after doing that, I realized that it was a lot of copy-paste so I wanted to minimize all that. After if in the future, the programs have more differences with the files (with new things to display, etc). Of course, I think that separating the display of programs and files is essential and you don't just have to look at that for the moment the only difference is the type.

For the ``DisplayFileCards.tsx`` file, it's the one attached to the sidebar and the tabs. So I was just thinking of adding ``ProgramCards`` to the program index and changing the filename to something like : ``DisplayContentsCards.tsx`` or ``DisplayCards.tsx``